packages/loaders/src/utils/useCSSSVGAnimation.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useEffect, useState } from 'react';
+
+export const useCSSSVGAnimation = () => {
+  const [canAnimateSVG, setAnimateSVG] = useState(true);
+
+  useEffect(() => {
```

Comment:
Is this truly a side-effect? If this value doesn't change or update it might make more sense as a constant that can be imported than as a hook.

Reply:
It needs to mutate the dom and then query it so I'd say yes